client/actions/simulations.js

```diff
@@ -0,0 +1,35 @@
+import * as types from './types';
+import api from '../api';
+
+export function simulationsHasErrored(hasErrored) {
+  return {
+    type: types.SIMULATIONS_HAS_ERRORED,
+    hasErrored: hasErrored
```

Comment:
You can use the notation without quotes for JavaScript Object since it's not pure JSON.